src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -61,10 +62,11 @@ public AddCommand parse(String args) throws ParseException {
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
         Set<Appointment> appointmentList = Collections.emptySet();
-
+        Priority priority = new Priority("NONE");
```

Comment:
You can consider adding a default constructor `public Priority()` that automatically creates a `Priority` with `NONE`

Reply:
Amended